tests/streaming/test_streaming_conversation.py

```diff
@@ -253,7 +257,8 @@ async def test_transcriptions_worker_ignores_utterances_before_initial_message(
             is_final=True,
         ),
     )
-    assert await _consume_worker_output(streaming_conversation.transcriptions_worker) is None
+    streaming_conversation.transcriptions_worker.start()
```

Comment:
Does start also need to be called before the `consume_nonblocking` call?

Reply:
no, events get processed after `start` is called